src/Discovery/Discovery.tsx

```diff
@@ -406,25 +450,82 @@ const Discovery: React.FunctionComponent<DiscoveryBetaProps> = (props: Discovery
               allowClear
             />
         }
-        { config.features.authorization.enabled &&
-            <div className='disvovery-table__controls'>
-              <Radio.Group
-                onChange={handleAccessLevelChange}
-                value={accessLevel}
-                className='discovery-access-selector'
-                defaultValue='both'
-                buttonStyle='solid'
+        { (
+          config.features.exportToWorkspaceBETA && config.features.exportToWorkspaceBETA.enabled
+        ) &&
+          <Space>
+            <span className='discovery-export__selected-ct'>{selectedResources.length} selected</span>
+            { config.features.exportToWorkspaceBETA.enableDownloadManifest &&
+              <Popover
+                className='discovery-popover'
+                arrowPointAtCenter
+                title={<>
+                  Download a Manifest File for use with the&nbsp;
+                  <a target='_blank' rel='noreferrer' href='https://gen3.org/resources/user/gen3-client/' >
+                    {'Gen3 Client'}
+                  </a>.
+                </>}
+                content={(<span className='discovery-popover__text'>With the Manifest File, you can use the Gen3 Client
+                to download the data from the selected studies to your local computer.</span>)}
               >
-                <Radio.Button value={AccessLevel.BOTH}>All</Radio.Button>
-                <Radio.Button value={AccessLevel.UNACCESSIBLE}><LockFilled /></Radio.Button>
-                <Radio.Button value={AccessLevel.ACCESSIBLE}><UnlockOutlined /></Radio.Button>
-              </Radio.Group>
-            </div>
+                <Button
+                  onClick={handleDownloadManifestClick}
+                  type='text'
```

Comment:
I'm curious if this is an arbitrary design choice? Personally I feel the look is a bit weird if this button has no border but the one next to it has

Reply:
The intention was to de-emphasize the button! We're actually planning a cleanup / style update of the Disco page for HEAL so I can take a second look at the button style then.